data/python.gram

```diff
@@ -488,7 +488,7 @@ statement_newline[list]:
     | a=compound_stmt NEWLINE { [a] }
     | simple_stmts
     | NEWLINE { [ast.Pass(LOCATIONS)] }
-    | ENDMARKER { None }
+    | ENDMARKER { [] }
```

Comment:
Why this change was made?

Reply:
None is now a valid result but we expect a list to be returned there.